reference/activity_manager/project_completeness.rst

```diff
@@ -0,0 +1,111 @@
+What is the project completeness?
+=================================
+
+Overview
+________
+
+In this reference guide you will learn what the project completeness refers to and how we manage it in Akeneo PIM. It
+will explain our technical and structure choices.
+
+Goal
+____
+
+The Activity Manager feature is about creating projects from a products selection that gives a vision on the enrichment
+progress to contributors (users in charge of the enrichment). Each project gives to the contributor its own completeness
+on the products selection, taking into account its permissions to edit or not products, attribute groups, locales and
+channels.
+
+For example, if Bob can only edit the "Technical" attribute group of the "High-tech" category, he will see the project
+100% complete if all attributes in the attribute group "Technical" of "High-tech" products in the project are filled in,
+even if other attribute groups are empty and required by the family. However, Julia, the owner of the project, will see
+that it is not complete if some attributes are not filled in others attribute groups. The contributor sees the
```

Comment:
_filled in in other attribute groups_ I know it sounds weird, but it's the right way